adserver/decisionengine/backends.py

```diff
@@ -32,6 +33,10 @@ def __init__(self, request, placements, publisher, **kwargs):
         :param placements: possible positions for the ad to go
         :param kwargs: Any additional possible arguments for the backend
         """
+        # Use the global pacing
+        # This spaces the flight over the whole flight by interval
+        self.interval = settings.ADSERVER_PACING_INTERVAL
```

Comment:
We should presumably make this configurable by flight, right? And make the setting a default? I don't think everyone will want it to be the same.

That also makes the deploy less scary, because we can just change 1 flight to the new interval.